scRNA-seq/02-filtering_scRNA.Rmd

```diff
@@ -279,60 +282,33 @@ num_genes_exp <- colSums(detection_mat)
 
 Let's plot this using the same style and type of graph as above. 
 
-```{r genes_expressed_plot, live = TRUE}
+```{r genes_expressed_plot}
 ggplot(mapping = aes(x = num_genes_exp)) +
   geom_density(fill = "lightblue") +  
   xlab("Number of genes expressed") +
   theme_classic() 
 ```
 
+This plot helps us visualize the distribution of genes per cell and can help inform how we choose the cutoff.
+We want to remove cells with a low number of genes per cell, but not be too strict that we lose a large portion of cells from the dataset. 
```

Comment:
Probably worth adding a comment here that summarizes the earlier content: we don't want to be strict partly because of the loss of data in general, but also because we are likely to lose a biased set of cells: If cell types vary in the number of genes expressed, we need to be extra careful!

Reply:
Definitely a good point. I modified some of the description, taking some of the text from the previous version and added it back in here.